Pyrado/pyrado/exploration/uniform_noise.py

```diff
@@ -87,7 +87,7 @@ def __init__(self,
             self.mean = None
 
         # Initialize parameters
-        self.log_halfspan_init = to.log(to.tensor(halfspan_init)) if isinstance(halfspan_init, float) else to.log(
+        self.log_halfspan_init = to.log(to.tensor(float(halfspan_init))) if isinstance(halfspan_init, (int, float)) else to.log(
```

Comment:
This will work, but using` to.tensor(std_min, dtype=to.get_default_dtype())` seems cleaner to me. However, if you dont want to change it, I am also fine with this solution.

Reply:
Done